src/Options/Aks/BaseAksOptions.cs

```diff
@@ -0,0 +1,11 @@
+using System.Text.Json.Serialization;
+using AzureMcp.Models.Option;
+using AzureMcp.Options;
+
+namespace AzureMcp.Options.Aks;
+
+public class BaseAksOptions : SubscriptionOptions
+{
+    [JsonPropertyName(OptionDefinitions.Common.ResourceGroupName)]
+    public new string? ResourceGroup { get; set; }
```

Comment:
This is defined in the very root Options class "GlobalOptions".  Do you need to redefine it?